src/features/movies/moviesSlice.js

```diff
@@ -3,27 +3,29 @@ import { createSlice } from "@reduxjs/toolkit";
 const moviesSlice = createSlice({
   name: "movies",
   initialState: {
-    loading: false,
+    loading: true,
```

Comment:
Trzeba przypisać do loading true na początku? Bez tego nie działa poprawnie apka? Bo ja zawsze daje false, bo de fakto w momencie powstania tego stanu nic nie wczytujemy, dopiero jak idzie fetch danych to wtedy jest loading ustawiany na true.

Reply:
zrobiłem tak, bo są wtedy niepotrzebne rendery na początku, tzn - nie renderuje się nic bo movies jest puste, potem jest trwa ładowanie i dopiero renderują się kafelki. Tak unikamy pierwszego kroku. A nawet jeżeli dasz na false to i tak nic Ci się nie pokaże bo nasz initialState jest pusty